.github/workflows/main.yml

```diff
@@ -18,9 +18,11 @@ jobs:
       matrix:
         node-version: [14.x]
     steps:
-    - uses: actions/checkout@v2
+      # https://github.com/actions/checkout/releases/tag/v4.0.0
+    - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac
     - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
+      # https://github.com/actions/setup-node/releases/tag/v3.8.1
+      uses: actions/setup-node@5e21ff4d9bc1a8cf6de233a3057d20ec6b3fb69d
```

Comment:
Upgraded from `v1` to `v3` because this new version has already been tried and tested on `e2e.yml`

https://github.com/HathorNetwork/hathor-wallet/blob/503d91c701e1a98a358c7a11a426f16cd2a5d0b2/.github/workflows/e2e.yml#L24-L25